### PR TITLE
Add documentation for a VI created to update the radar plot VI.

### PR DESCRIPTION
**Pull request by citlaligm**
## Overview
Creates documentation for one VI added to the LV client.

## Changes
- Add the documentation for `mono_get_radar_config_params.md`

## Test
This is the corresponding build https://monodrive.readthedocs.io/en/radar_plot_update/LV_client/tools/mono_matlab_radar_plot/

### CODE REVIEW

File: docs/LV_client/tools/mono_matlab_radar_plot.md [hunk 1]
```diff
@@ -6,16 +6,22 @@
...
 
-Process Radar Data Cube and run a Matlab script to plot Radar data.
+Process radar data cube and run a Matlab script to plot Radar cube data.
 
 For technical support contact us at <b>support@monodrive.io</b> 
 
 ### Inputs
 
-- **Radar data cube:**  Data cube from Radar ouput 
+- **Radar Data Cube:**  Only send if, **send_cube** is set to true on the **Radar
```
Comment (devinaconley) [inline docs/LV_client/tools/mono_matlab_radar_plot.md:15]: Should note somewhere that `"send_radar_cube"` must be true for this plotter to work

**Review by devinaconley — CHANGES_REQUESTED** [review]
Looks good! One small suggestion

File: docs/LV_client/tools/mono_matlab_radar_plot.md (hunk 1 above)
Comment (citlaligm) [reply docs/LV_client/tools/mono_matlab_radar_plot.md:15]: yeah, good catch!

File: docs/LV_client/tools/mono_matlab_radar_plot.md [hunk 2]
```diff
@@ -6,16 +6,24 @@
...
```
Comment (devinaconley) [inline docs/LV_client/tools/mono_matlab_radar_plot.md:15]: I think this field is called `"send_radar_cube"`

File: docs/LV_client/tools/mono_matlab_radar_plot.md [hunk 3]
```diff
@@ -6,16 +6,24 @@
...
+configuration**. Raw data for a cube of dimensions M (number
+of array antenna inputs) by L (number of range bins in fast
+time) by N (number of pulse in CPI in slow time).
+
+**NOTE:** If the Radar data cube is empty (**send_cube** is set to false) the plot will be empty or won't show.
```
Comment (devinaconley) [inline docs/LV_client/tools/mono_matlab_radar_plot.md:20]: same comment

**Review by devinaconley — CHANGES_REQUESTED** [review]
Just a quick variable name fix

File: docs/LV_client/utilities/mono_get_radar_config_params.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+# mono_get_radar_config_params.vi
+Parses the configuration an obtain the following parameters:  elements, the number of sweeps  and  number of samples per sweep.
```
Comment (katiemacaulay) [inline docs/LV_client/utilities/mono_get_radar_config_params.md:2]: Do you mean "Parses the configuration **and obtains**"?

File: docs/LV_client/tools/mono_matlab_radar_plot.md [hunk 5]
```diff
@@ -6,16 +6,24 @@
 
 ### Description
 
-Process Radar Data Cube and run a Matlab script to plot Radar data.
+Process radar data cube and run a Matlab script to plot Radar cube data.
```
Comment (katiemacaulay) [inline docs/LV_client/tools/mono_matlab_radar_plot.md:9]: Should radar be Capitalized in both instances?

**Review by katiemacaulay — COMMENTED** [review]
@citlaligm, looking good. Just a couple of questions/suggestions.

**Comment by citlaligm** [comment]
@katiemacaulay thanks for the feedback should be ready for another look.

**Review by katiemacaulay — APPROVED** [review]
@citlaligm, looks good to go.

**Pull request merged**